include/wx/localedefs.h

```diff
@@ -82,12 +82,15 @@ enum wxLocaleInfo
 struct WXDLLIMPEXP_BASE wxLanguageInfo
 {
     int Language;                   // wxLanguage id
+    wxString LocaleTag;             // Tag of locale in BCP 47-like notation
     wxString CanonicalName;         // Canonical name, e.g. fr_FR
+    wxString CanonicalRef;          // Canonical name reference, e.g. fr_FR for fr
```

Comment:
In principle, I'm fine with your suggested wording.

However, I added `CanonicalRef` only for those entries of the previous language database, for which no entry with just the language code existed. I did that, so that the translation loading method can be adjusted to look not only in the directory specified by the language code alone (for example **fr**), but also in the directory specified by language and region (for example **fr_FR**).

I'm not sure whether we really need to do that, but I made these provisions to be on the safe side. BTW, the change to the translation loading code has still to be done.